Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -362,7 +362,11 @@ def _boot_device(device, host=None):
         host = host or SystemHost.get_default()
         _log.debug(u"Booting device '{}'".format(device.udid))
         device.platform_device.booted_by_script = True
-        host.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'boot', device.udid])
+        try:
+            host.executive.run_command([SimulatedDeviceManager.xcrun, 'simctl', 'boot', device.udid])
+        except ScriptError as e:
+            _log.error('Error: ' + e.message_with_output(output_limit=None))
+            exit(e.exit_code or 2)
```

Comment:
We were raising this error before. We should `raise` on line `369` instead of a specific exit. That's actually much better for callers in `webkitpy` than doing a `sys.exit` (which throws an exception anyways, just one that's harder to handle) because callers can (and do) manage exceptions.

Reply:
Great point, I just changed that. Thank you!